packages/core/src/client.ts

```diff
@@ -172,6 +172,15 @@ export interface MedplumClientOptions {
    */
   fetch?: FetchLike;
 
+  /**
+   * The profile name.
+   *
+   * This is for handling multiple profiles to access different FHIR Servers
+   *
+   * If undefined, the default profile is "default".
+   */
+  profile?: string;
```

Comment:
Reason I had it as another parameter in createMedplumClient was because it was of MedplumClientOptions, and didn't think it was good to continue adding on to the type. After your review I added it here